.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,44 @@
+name: 🐞 Bug report
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
```

Comment:
we could try encouraging people to first start a "Discussion" and if needed - open an issue.

Reply:
The discussions are not enabled for this repository, yet.